semantic_conventions/logs/log-general.yaml

```
@@ -0,0 +1,17 @@
+groups:
+  - id: log.record
+    prefix: log.record
+    brief: >
+      The attributes described in this section are rather generic. They may be used in any Log Record they apply to.
+    attributes:
+      - id: id
+        type: string
+        requirement_level: recommended
+        brief: >
+          A unique identifier for the Log Record.
+        note: >
+          MUST be an [Universally Unique Lexicographically Sortable Identifier (ULID)](https://github.com/ulid/spec).
+          If a value is provided other log records, with the same value, will be considered to be duplicates and can be removed safely.
+          This means, that two distinguishable log records MUST have different values.
+          However in the case of duplication it is not guaranteed that duplicated log records have the same value.
+        examples: ["click", "exception"]
```

```suggestion
        examples: ["01ARZ3NDEKTSV4RRFFQ69G5FAV"]
```
The examples are wrong, right?

---

yeah, I thought I replaced that, not sure what happened here.